app/views/event_steps/_personalised_updates.html.erb

```diff
@@ -11,7 +11,7 @@
 <%= f.govuk_collection_select :degree_status_id,
       f.object.degree_status_options,
       :id,
-      ->(option) { degree_status_text(option) },
+      ->(option) { format_option_value(option, "mailing_list_steps.teacher_training.degree_status") },
```

Comment:
should this be `event_steps` not `mailing_list_steps`?

Reply:
Ah I believe I'm just reusing the front end text from the mailing_list_steps translation here, it should read the same in this section as it does in the mailing list